DataFormats/PatCandidates/interface/MET.h

```diff
@@ -140,28 +149,57 @@ namespace pat {
 
       // ---- members for MET corrections ----
       enum METUncertainty {
-        JetEnUp=0, JetEnDown=1, JetResUp=2, JetResDown=3,
+	JetResUp=0, JetResDown=1, JetEnUp=2, JetEnDown=3,
```

Comment:
The swap is indeed artificial, but makes the handling of the jet smeared uncertainty case much easier in the pure 75X release. As it was much too late to backport the swap in 74X, that small inconsistency exists.

If people follows the MET recommendations, the results will be the same anyway, as the analyzers are supposed to vary both JES and JER. So both uncertainties will be taken into account anyway.